sites/partners/src/lib/hooks.ts

```diff
@@ -612,3 +616,16 @@ export function useMapLayersList(jurisdictionId?: string) {
     mapLayersError: error,
   }
 }
+
+export function useLotteryActivityLog(listingId: string) {
+  const { lotteryService } = useContext(AuthContext)
+  const fetcher = () => lotteryService.lotteryActivityLog({ id: listingId })
+
+  const { data, error } = useSWR(`/api/adapter/lottery/lotteryActivityLog`, fetcher)
```

Comment:
This doesn't have to be done as part of this PR, but SWR provides a lot of options so that we can reduce the number of times it calls the backend. For example right now on this call if you switch to a different tab but don't close it and then go back to the page it will re-call this page. We could remove that call by adding the `revalidateOnFocus: false` value here. 

However, we probably should discuss as a team if that is a pattern we want since it's not really hurting anything right now